### ISSUE TITLE
CDC COVID-19 - Bug: Dates text in the "Latest Updates" section does not meet color contrast requirements

### ISSUE THREAD

**Issue by tiffanyjc** [a11ySea/testing#7]
## Issue Summary

The text containing the date in the "Latest Updates" section on the CDC page has insufficient color contrast (3.97).

## Steps to reproduce

1. Go to https://www.cdc.gov/coronavirus/2019-ncov/index.html
2. Scroll down to the "Latest Updates" section
3. The running column of dates with the color #808080 does not meet color contrast requirements (but the associated links next to them are OK)

### Expected behavior

Expected contrast ratio for this text size is at least 4.5:1.

### Actual behavior

The dates text in the "Latest Updates" section on the CDC page has insufficient color contrast of 3.97 (foreground color: #808080, background color: #ffffff, font size: 15.0pt (20px), font weight: normal). Expected contrast ratio of 4.5:1. 

# Specifications
- **WCAG: WCAG 1.4.3**
